b2c/Create-AzureB2C.ps1

```diff
@@ -143,7 +148,7 @@ function New-AzureADB2CTenant {
 
   if (!$checkRg) {
     Write-Warning "Resource Group $AzureResourceGroup does not exist. Creating..."
-    az group create --name $AzureResourceGroup --location "northeurope" # Everybody likes Ireland, so we put the RG there if it does not exist
+    az group create --name $AzureResourceGroup --location $(($Location.ToLower()).replace(' ','')) # Everybody likes Ireland, so we put the RG there if it does not exist
```

Comment:
The `--location` parameter when creating a Resource Group must be an Azure region, which is different from the "location" of B2C tenant. That's why the `$Location` variable was not used here ("unitedstates", "europe", ... are not valid Azure regions).

Reply:
On second thought, using a logical region when none is provided, is probably better than hardcoding "northeurope". Would you mind updating the comment as well, to explain this?

Reply:
Hi Martine, I'm trying to run your scripts getting some errors ash shown below. Could you advice on this

![image](https://user-images.githubusercontent.com/25347487/231718823-df5c6854-f732-487d-878a-d08eba4dc0f9.png)